packages/fxa-auth-server/docs/swagger/descriptions.ts

```diff
@@ -0,0 +1,141 @@
+  limit:
+    'The maximum number of commands to return. The default and maximum value for limit is 100.',
+  location: "Object containing the client's state and country",
+  messages: 'An array of individual commands for the device to process.',
+  originalLoginEmail:
+    'This parameter is the original email used to login with. Typically, it is specified after a user logins with a different email case, or changed their primary email address.',
+  queryKeys:
+    'Indicates whether a new `keyFetchToken` is required, default to `false`.',
+  payload: 'Opaque payload to be forwarded to the device.',
+  ppidSeed:
+    'Seed used in sub claim generation of JWT access tokens/ID tokens for clients with [**Pseudonymous Pairwise Identifiers (PPID)**](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/oauth/pairwise-pseudonymous-identifiers.md) enabled. Used to forcibly rotate the `sub` claim. If not specified, it will default to `0`.',
+  publicKey:
+    'The key to sign (run bin/generate-keypair from [**browserid-crypto**](https://github.com/mozilla/browserid-crypto)).',
+  pushPayload:
+    'Push payload, validated against [**pushpayloads.schema.json**](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/pushpayloads.schema.json).',
+  reason:
+    'Alphanumeric string indicating the reason for establishing a new session; may be "login" (the default) or "reconnect".',
+  recoveryData: "An encrypted bundle containing the user's kB.",
+  recoveryKeyId:
+    'A unique identifier for this recovery key, derived from the key via HKDF.',
+  redirectTo:
+    'URL that the client should be redirected to after handling the request.',
+  redirectUri:
+    'The URI at which the connecting client expects to receive the authorization code. If supplied this must match the value provided during OAuth client registration.',
+  refreshToken:
+    'A token that can be used to grant a new access token when the current one expires, via `grant_type=refresh_token` on this endpoint.',
+  reminder: 'Indicates that verification originates from a reminder email.',
+  resource:
+    'Indicates the target service or resource at which access is being requested. Its value must be an absolute URI, and may include a query component but must not include a fragment component. Added to the `aud` claim of JWT access tokens.',
+  responseType:
+    "Determines the format of the response. Since we only support the authorization-code grant flow, the only permitted value is 'code'.",
+  resume:
+    'Opaque URL-encoded string to be included in the verification link as a query parameter.',
+  scope:
+    'A space-separated list of scope values that the connecting client will be granted. The requested scope will be provided by the connecting client as part of its authorization request, but may be pruned by the user in a confirmation dialog before being sent to this endpoint.',
+  scopeValue: 'The scope values held by the granted access token.',
```

Comment:
There were two descriptions for scope, do we want to use the first description in both places?

Reply:
What's the other description? If it's got more detail you could use it or combine them but it seems fine.

Reply:
The two descriptions for `scope` are:
(1) A space-separated list of scope values that the connecting client will be granted. The requested scope will be provided by the connecting client as part of its authorization request, but may be pruned by the user in a confirmation dialog before being sent to this endpoint.
(2) The scope values held by the granted access token.

The first is used in `GET /account/attached_clients`. The second is used `POST /oauth/token`.